tls/s2n_x509_validator.c

```diff
@@ -430,6 +430,11 @@ static S2N_RESULT s2n_x509_validator_verify_cert_chain(struct s2n_x509_validator
                 S2N_ERR_INTERNAL_LIBCRYPTO_ERROR);
     }
 
+#if defined(LIBRESSL_VERSION_NUMBER)
+    RESULT_GUARD_OSSL(X509_VERIFY_PARAM_clear_flags(param, X509_V_FLAG_CB_ISSUER_CHECK),
+            S2N_ERR_INTERNAL_LIBCRYPTO_ERROR);
+#endif
```

Comment:
I think this call to clear_flags should no longer be needed after https://github.com/aws/s2n-tls/pull/3671.

Reply:
Oh, nice. Thanks for pointing that out.